libnavui-base/src/main/java/com/mapbox/navigation/ui/base/util/BitmapEncodeOptions.kt

```diff
@@ -0,0 +1,87 @@
+package com.mapbox.navigation.ui.base.util
+
+import androidx.annotation.IntRange
+
+private const val DEFAULT_BITMAP_ENCODE_WIDTH = 250
```

Comment:
may move constants to `companion` of `BitmapEncodeOptions`?

Reply:
It is actually a bit more efficient this way, because with `companion object` an unnecessary instance of a `Companion` class is created.